Purchases/Error Handling/ErrorUtils.swift

```diff
@@ -347,4 +320,44 @@ private extension ErrorUtils {
         return errorWithUserInfo
     }
 
+    private static func logErrorIfNeeded(_ code: ErrorCode) {
+        switch code {
+        case .networkError,
+                .unknownError,
```

Comment:
formatting is weird, I wonder why they got pushed out like this

Reply:
Xcode auto-formatting was doing it like this, couldn't figure out why.

Reply:
Ugh, I hate it, but I think we should stick with whatever the default is for Xcode

Reply:
Yeah not worth indenting manually I think..